wonjong/services/postService.js

```diff
@@ -9,14 +8,22 @@ const createPost = async (title, content, userId) => {
 const allPostViews = async () => {
   return await postDao.allPostViews();
 };
-const patchPost = async (postId, title, content) => {
-  const createPost = await postDao.patchPost(postId, title, content);
-  return createPost;
+const patchPost = async (userId, postId, title, content) => {
```

Comment:
함수와 함수 사이에 공백 넣어주세요!!
```suggestion

const patchPost = async (userId, postId, title, content) => {
```